src/commands/help.js

```diff
@@ -0,0 +1,19 @@
+const Discord = require('discord.js');
+const logger = require('./../logger');
+
+module.exports = {
+	help : {
+		name : 'help',
+		args : false,
+		description : 'Command to call for help from the Discord bot',
+		execute: (message) => {
+			message.channel.send('',
+				new Discord.RichEmbed({description: '!f :KannaWave: - creates big emote \
+                                                                          \n !f avatar - sends big avatar of user\
+                                                                          \n !f whoami - sends username\
+                                                                          \n !f tft Jacob_Hong - sends tft profile ' }))
+				.then(() => logger.info('Help message relayed'))
```

Comment:
Should we dynamically expand the list like we did in the bot js? just in case there is a command change, we still need to manually updated it. Im fine with this for now, i can add the change later.

Reply:
I think that's a good idea - you should tackle it for your first task 👍